ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -732,6 +732,17 @@ tests:
         TEST_FUNC=test_networking
       PACKET_PLAN: m3.large.x86
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-bond
+  optional: true
+  run_if_changed: ^(data/.*)$
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        IS_BONDED=true
+        TEST_FUNC=test_networking
+      PACKET_PLAN: m3.large.x86
```

Comment:
```suggestion
```
we spawn a 3 masters/2workers cluster, right? In this case the default machine should work

Reply:
Actually I copied from STATIC_IP. It has the same configuration.  Do you know why ?

Reply:
you need to test on OVN and SDN? https://sourcegraph.com/github.com/openshift/assisted-test-infra/-/blob/src/tests/test_e2e_install.py?L21-23

If that's the case then we can let like this